x-pack/plugins/security_solution/public/management/pages/index.test.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { ManagementContainer } from './index';
+import { AppContextTestRender, createAppRootMockRenderer } from '../../common/mock/endpoint';
+import { useIngestEnabledCheck } from '../../common/hooks/endpoint/ingest_enabled';
+
+jest.mock('../../common/hooks/endpoint/ingest_enabled');
+
+describe('when in the Admistration tab', () => {
+  let render: () => ReturnType<AppContextTestRender['render']>;
+
+  beforeEach(() => {
+    const mockedContext = createAppRootMockRenderer();
+    render = () => mockedContext.render(<ManagementContainer />);
+  });
+
+  it('should display the No Permissions view when Ingest is OFF', async () => {
+    (useIngestEnabledCheck as jest.Mock).mockReturnValue({ allEnabled: false });
```

Comment:
Why do you need to cast the `useIngestEnabledCheck`? You should not have to, so I'm thinking something in the mocked interface is not right.

Reply:
you're right that it's not required, was just following other examples - will remove

Reply:
I take that back, I remember why now - the tests will run correctly without it, but it fails typecheck

![image](https://user-images.githubusercontent.com/56395104/86943310-476c2d80-c114-11ea-9090-52b7def84401.png)